test/src/test/java/hudson/model/SimpleJobTest.java

```diff
@@ -19,20 +19,20 @@
 
     @Rule
     public JenkinsRule rule = new JenkinsRule();
-    
+
```

Comment:
This file contains no changes other than whitespace. Please revert, and configure your IDE to leave lines alone which you aren't editing.

Reply:
No idea how that one got in there. I didn't intend to make any changes and my IDE isn't set to automatically make those changes.